graphql/batch_executor.go

```diff
@@ -80,6 +89,7 @@ func (e *Executor) Execute(ctx context.Context, typ Type, source interface{}, qu
 	topLevelRespWriter := newTopLevelOutputNode(query.Name)
 	initialSelectionWorkUnits := make([]*WorkUnit, 0, len(topLevelSelections))
 	writers := make(map[string]*outputNode)
+	ctx = context.WithValue(ctx, "pagiantionScheduler", e.scheduler)
```

Comment:
nit: small typo here "pagiantionScheduler" -> "paginationScheduler"
Also, consider making this string a `const` to ensure that all callers are using the exact same string.